sdk/python/kfp/containers/_build_image_api.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the speci
+
+__all__ = [
+    'build_image_from_working_dir',
+    'default_base_image',
+    'get_python_image_for_current_version',
+]
+
+
+import logging
+import os
+import re
+import shutil
+import sys
+import tempfile
+
+import requests
+
+from . import get_default_image_builder
+from ..compiler._container_builder import ContainerBuilder
+
+
+def get_python_image_for_current_version() -> str:
```

Comment:
Remove

Reply:
Do you think that users who want their images to be smaller could benefit from this?

Reply:
Removed this user convenience function.